src/components/simpleHeader/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react';
+
+import { chakra, Flex, useMultiStyleConfig } from '@chakra-ui/react';
+
+import { CloseIcon } from '../../index';
+
+interface Props {
+  title: string;
+  url: string;
+}
+
+const SimpleHeader: FC<Props> = ({ title, url }) => {
+  const styles = useMultiStyleConfig(`SimpleHeader`);
+
+  const Component = chakra('header', { baseStyle: styles.header });
+
+  return (
+    <Component>
+      <Flex
+        minH={{ xs: 'xl', lg: '2xl' }}
+        paddingX={{ xs: 'md', lg: '4xl' }}
```

Comment:
Adding the spacing to the header feels a bit wired from a layout perspective to me. this means we will have to make sore to add the same spacing on the page content instead of a layout that defines the spacing between the page and the container.

Reply:
Good point!
Maybe I can remove those padding to have control on the layout.
I will open a PR for that change.
Thanks! 🧸